src/AddTodoForm.jsx

```diff
@@ -1,26 +1,35 @@
 import { useState } from 'react';
+import InputWithLabel from './InputWithLabel';
 
 function AddTodoForm({ onAddTodo }) {
-  const [todoTitle, setTodoTitle] = useState('');
+	const [todoTitle, setTodoTitle] = useState('');
 
-  const handleTitleChange = event => {
-    const newTodoTitle = event.target.value;
-    setTodoTitle(newTodoTitle);
-  };
+	const handleTitleChange = event => {
+		const newTodoTitle = event.target.value;
+		setTodoTitle(newTodoTitle);
+	};
 
-  const handleAddTodo = event => {
-    event.preventDefault();
-    onAddTodo(todoTitle);
-    setTodoTitle('');
-  };
+	const handleAddTodo = event => {
+		event.preventDefault();
+		if (todoTitle !== '') {
```

Comment:
It's a good idea to prevent empty todo titles. The main drawback to this approach is that it doesn't indicate to the user that they cannot hit submit. Another way to block a submit would be to disable the button if the todoTitle's length was 0.